main.py

```diff
@@ -47,7 +47,21 @@ def main():
                     print("There is no file.")
             case "11":
                 break
-
+            case "1":
+                try:
+                    view_all_trans(df)
+                except UnboundLocalError or ValueError:
+                    print("There is no file.")
+            case "2":
+                try:
+                    data_management.view_trans_by_date_range(df)
+                except UnboundLocalError or ValueError:
+                    print("There is no file.")
+            case "6":
+                try:
+                    data_management.analyze_spending_by_category(df)
+                except UnboundLocalError or ValueError:
+                    print("There is no file.")
```

Comment:
From line 38 to line 47 and line 50 to line 59 are duplicated.
Maybe you accepted both changes when resolving conflicts?